### DISCUSSION TITLE
Enable category permission settings

### DISCUSSION THREAD

**Discussion by FillingTheVoidOnYT** [stoatchat/discussions#436] [Feature Suggestions]
Having the ability to just add permissions to a certain category then syncing all the newly created channel to the category would be so nice to have.

**Comment by oscoDOTblog** [comment]
bumping this, really would love this feature!!!

for me, it's the primary reason i'm sticking to discord as i manage a server with lots of different categories and it makes it super easy to manage the channels that way.

**Comment by aurorapar** [comment]
Bumping this
